doc/quick-start/quick-start.md

```diff
@@ -1,5 +1,39 @@
 # Quick Start for Data Prep Kit 
-Here we provided short examples of various uses of the Data Prep Kit.
+Here we provided short examples of various uses of the Data Prep Kit. Most users who want to jump right in can use standard pip install to deploy the data-prep-kit and the python or ray transforms to their virtual python environment. 
```

Comment:
Can we have a create-venv.md that breaks this out and then has separate sections for conda and venv.  It would also be good to have a link to install conda.  I believe venv comes with the base python install so no link needed there.

```
python -m venv venv
source venv/bin/activate
pip install ...
...
deactivate
```

Reply:
@daw3rd Let's have that as a separate issue/PR to revisit the documentation. As a new comer, and please I don't want anyone to take this the wrong way as I know a lot of effort and thoughts went into it, but all those different MD we have are quite confusing to navigate and increase difficult for maintaining it all. I don't know if adding one more will help or hurt. Maybe neither but there is a point to be made about producing the right amount of documentation that meets most users need at this point and iterate **as our users** (emphasis on users) ask for it.

Reply:
I just did this and tested on both conda and venv environments,

https://github.com/sujee/data-prep-kit/blob/rag-example1/examples/notebooks/rag/setup-python-dev-env.md

happy to submit a PR for dev-env setup, if needed.

Reply:
@touma-I Yep this is a style thing.  I personally do not like navigating through a long document to find the content I want and find it much easier to have a page full of links to focused content.